lib/manager/gomod/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -275,15 +275,19 @@ Array [
     },
   },
   Object {
-    "cmd": "docker run --rm --name=renovate_go --label=renovate_child -v \\"/tmp/github/some/repo\\":\\"/tmp/github/some/repo\\" -v \\"/tmp/renovate/cache\\":\\"/tmp/renovate/cache\\" -e GOPROXY -e GOPRIVATE -e GONOPROXY -e GONOSUMDB -e GOFLAGS -e CGO_ENABLED -e GIT_CONFIG_KEY_0 -e GIT_CONFIG_VALUE_0 -e GIT_CONFIG_COUNT -w \\"/tmp/github/some/repo\\" renovate/go:latest bash -l -c \\"go get -d ./...\\"",
+    "cmd": "docker run --rm --name=renovate_go --label=renovate_child -v \\"/tmp/github/some/repo\\":\\"/tmp/github/some/repo\\" -v \\"/tmp/renovate/cache\\":\\"/tmp/renovate/cache\\" -e GOPROXY -e GOPRIVATE -e GONOPROXY -e GONOSUMDB -e GOFLAGS -e CGO_ENABLED -e GIT_CONFIG_KEY_0 -e GIT_CONFIG_VALUE_0 -e GIT_CONFIG_KEY_1 -e GIT_CONFIG_VALUE_1 -e GIT_CONFIG_KEY_2 -e GIT_CONFIG_VALUE_2 -e GIT_CONFIG_COUNT -w \\"/tmp/github/some/repo\\" renovate/go:latest bash -l -c \\"go get -d ./...\\"",
     "options": Object {
       "cwd": "/tmp/github/some/repo",
       "encoding": "utf-8",
       "env": Object {
         "CGO_ENABLED": "1",
-        "GIT_CONFIG_COUNT": "1",
-        "GIT_CONFIG_KEY_0": "url.https://some-token@github.com/.insteadOf",
+        "GIT_CONFIG_COUNT": "3",
+        "GIT_CONFIG_KEY_0": "url.\\\"https://api:some-token@github.com/\\\".insteadOf",
```

Comment:
Is this change from `some-token@` to `api:some-token@` intentional?

Reply:
Is the `url.\\\"` a bug fix, which we perhaps apply separately? And I'm assuming it definitely works?!

Reply:
The change is intentiomal, so that you can distinguish in the log what cause the request. Also from the page:
```
Note: the "usernames" are arbitrary, but must be uinque, as the insteadOf syntax is reversed from what you might expect: the replacement string is key, and the pattern string is the value.
```
So we need three different urls, otherwise we cannot support all three occurences.
That reminds me, that for this to work with gitlab I need to think about something else.
The username is fixed and therefore only one replacement is supported.

Reply:
I orientated myself on the guide, so that the url is clearly separated to the surrounding context. It works with and without, but I can imagine that without `"` we could run into edge cases.

I can live with both.

Reply:
Let's keep it, but I was thinking whether to separate it out into its own PR

Reply:
> Note: the "usernames" are arbitrary, but must be uinque, as the insteadOf syntax is reversed from what you might expect: the replacement string is key, and the pattern string is the value.

That's very unintuitive!

> That reminds me, that for this to work with gitlab I need to think about something else.

Surely we're not the first to have this problem. Finding a solution sounds challenging though